### DISCUSSION TITLE
单独页面怎么实现js交互

### DISCUSSION THREAD

**Discussion by krvipsoft** [slowlyo/owl-admin#123] [FAQ]
dcat admin 是直接可以写在方法里直接加载，比如 Admin::script($this->scripts());
文档里写 仅支持全局加载，还有其他方式实现吗？因为有些功能只是针对当前页面的

**Comment by slowlyo** [comment]
没有，owl admin是单页应用，只加载一次静态资源

**Comment by krvipsoft** [comment]
好吧，只能自己研究amis
